apps/src/javalab/JavalabEditor.jsx

```diff
@@ -360,6 +362,40 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  // This moves the active tab to the left in the tab menu
+  moveTabLeft() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
```

Comment:
nit: this is pretty minor, but is there any reason to clone and create `newTabs` here rather than inside `swapTabsSetKeyCloseTab`? Unless we plan on using `swapTabsSetKeyCloseTab` to swap tabs from a different set other than ordered tab keys, I feel like it might more sense to clone the array inside the helper function. That way you only have to pass is `index1, index2`

Reply:
Thanks! This makes sense - will do.